scripts/setup.py

```diff
@@ -143,6 +143,12 @@ def download_and_install_node() -> None:
                 node_file_name = 'node-v%s-darwin-x64' % (common.NODE_VERSION)
             elif common.is_linux_os():
                 node_file_name = 'node-v%s-linux-x64' % (common.NODE_VERSION)
+            # Oppia only suppports windows, mac and linux operating systems.
```

Comment:
I don't think this works. Adding the exception case is correct, but you should test it by mocking is_mac_os(), is_linux_os(), etc. to return False.

(Note that basically any time I see "# pragma: no cover", I will probably ask for its removal -- in most cases the test can be done.)

Reply:
Added a new test removing this flag.